src/IncludedCommands/UsefulFun/fly.lua

```diff
@@ -247,42 +267,101 @@ return {
                         for _,Event in pairs(self.Events) do
                             Event:Disconnect()
                         end
+
+                        --Unbind the ContextActionService bind
+                        ContextActionService:UnbindAction("toggleFlight")
     
                         --Unstore the flight.
                         if Api.CommandData.CurrentFlight == self then
                             Api.CommandData.CurrentFlight = nil
                         end
                     end
+
+                    ContextActionService:BindAction("toggleFlight",function(actionName, UserInputState, InputObject)
```

Comment:
Camel case isn't used in the project, and there should be a space after the comma. Requested line:
```suggestion
                    ContextActionService:BindAction("ToggleFlight", function(ActionName, UserInputState, InputObject)
```

(Also remember to change `toggleFlight` to `ToggleFlight` in other cases)